README.md

```diff
@@ -1 +1,171 @@
-# testing-to-do-list
+# To-Do-List
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 To Do List <a name="about-project"></a>
+
+**To Do List** is a website for listing different 'to do' activities.
+The website is build with HTML, CSS and Javascript.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css">CSS</a></li>
+    <li><a href="https://www.w3schools.com/js">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Add New Book to the list**
+- **Remove Book from the list**
+- **Saves Data into localStorage**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://wilhelmk109.github.io/To-Do-List/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- NPM machine on your machine
+- A browser of your choice.
+- Node.js environment
+
+### Setup
+
+Clone this repository to your desired folder by running the below command in your terminal:
+
+- `git clone git@github.com:WilhelmK109/testing-to-do-list.git`
+
+### Install
+
+Install this project with command: npm i
+
+### Usage
+
+To run the project, execute the following command: npm run server
+
+
+### Run tests
+
+To run tests, run the following command:
+
+
```

Comment:
- [ ] This milestone being a testing one, it can be good if a command for running the tests is available here for users so that it can be easy for them(us).